.gitignore

```diff
@@ -34,4 +34,7 @@ build/
 .vscode/
 
 ### Environment ###
-*.env
\ No newline at end of file
+*.env
+
+### Logs ###
```

Comment:
im not sure this is working as intended, given the fact that application.log got added to the merge request?

Reply:
I believe this happened because when I created the PR I had the ignore file set to */log or something like that. Since then I have updated the code and pushed it again. It should be working now, but I could just create a new PR to see if that is true?

Reply:
nah i can just remove the file from commit before I merge in that case

Reply:
k. If I did it wrong, we'll for sure know the next time someone makes a PR after pulling the latest from main XD

Reply:
nah i tested it on my end, once I remove that file from vc, the whole folder gets ignored. that file was just added to vc before the gitignore got updated most likely?